dom/nodes/Document-createEvent.html

```diff
@@ -99,48 +101,57 @@
   "IDBVersionChangeEvent",
   "InstallEvent",
   "KeyEvent",
-  "MIDIConnectionEvent",
-  "MIDIMessageEvent",
+  "KeyEvents",
   "MediaEncryptedEvent",
   "MediaKeyEvent",
   "MediaKeyMessageEvent",
   "MediaQueryListEvent",
   "MediaStreamEvent",
   "MediaStreamTrackEvent",
+  "MIDIConnectionEvent",
+  "MIDIMessageEvent",
+  "MouseScrollEvents",
```

Comment:
For this and PopUpEvents, maybe list them as non-plural so that "MouseScrollEventss" and "PopUpEventss" aren't tested?